pkg/exchange/okex/exchange.go

```diff
@@ -411,6 +416,124 @@ func (e *Exchange) QueryOrderTrades(ctx context.Context, q types.OrderQuery) ([]
 	if errs != nil {
 		return nil, errs
 	}
+	return trades, nil
+}
+
+/*
+Query Closed trades can query closed trades in last 3 months, there are no time interval limitations, as long as until >= since.
+Please Use LastTradeID as cursor, only return trades earlier than that trade, that trade is not included.
+If you want to query trades by time range, please just pass since and until.
+If you want to query by cursor, please pass LatestTradeID.
+Because it get correct response even pass all parameters with right time interval and invalid trade id, like 0.
+*/
+func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, since, until time.Time, lastOrderID uint64) ([]types.Order, error) {
+	if symbol == "" {
+		return nil, ErrSymbolRequired
+	}
+
+	if err := tradeRateLimiter.Wait(ctx); err != nil {
+		return nil, fmt.Errorf("query closed order rate limiter wait error: %w", err)
+	}
+
+	var lastOrder string = strconv.FormatUint(lastOrderID, 10)
+
+	res, err := e.client.NewGetOrderHistoryRequest().
+		InstrumentID(toLocalSymbol(symbol)).
+		StartTime(since).
+		EndTime(until).
+		Limit(defaultQueryLimit).
+		After(lastOrder).
```

Comment:
I think we should use `Before` instead of `After`. The order ID is generally an incrementing number. Am i right @c9s ?

Reply:
refer: https://www.okx.com/docs-v5/en/#order-book-trading-trade-get-order-history-last-3-months